sway-core/src/abi_generation/abi_str.rs

```diff
@@ -208,6 +211,19 @@ impl TypeInfo {
     }
 }
 
+fn handle_abi_name_attribute(call_path: &CallPath, attributes: &transform::Attributes) -> CallPath {
```

Comment:
Maybe `get_abi_call_path`? Sounds to me more like the intention, whereas the "handling of attributes" is an implementation detail.